spec/batching_job_spec.rb

```diff
@@ -59,15 +59,6 @@ def on_success(_, options)
       end
     end
 
-    it "runs the on_success callback even if no batches are run" do
```

Comment:
did we make a mistake in the other MR? I feel like these callbacks should be called if no batches run but we expected sidekiq to run them and not simplekiq

Reply:
Yea I think you're correct here, though I know sidekiq batches don't run when using `inline!`. The spec helper in this library also calls this out. These _may_ have only been successful due to the empty batch work the library was previously handling. 

I can mark these as pending to get the action work in, and we can revisit next week?

Reply:
or maybe we need to update these tests to assert against the sidekiq batch and not the simplekiq batch

Reply:
> I can mark these as pending to get the action work in, and we can revisit next week?

yeah I don't think it's urgent to get these tests to pass

Reply:
Hmm, I agree with Farjaad that these tests *ought* to pass? I "missed" them before because they were failing due to the lack of redis and I didn't want to try to tackle that as well.

But yeah I think this is fine for now.